### ISSUE TITLE
Unable to replicate the quality of captions for Visual Genome Dataset

### ISSUE THREAD

**Issue by SuhasShanbhogue** [ruotianluo/ImageCaptioning.pytorch#108]
We trained the model for 50 epochs (using the Non- self Critical training).
The loss converges very high and the captions are terrible grammatically with lot of repeated phrases.
Are we doing some trivial mistake?

**Comment by behome** [comment]
I also had the same problem. The loss converges to about 5.7 and stops, and the word a is repeatedly generated, which is the word with the highest frequency in the vocabulary.
